### ISSUE TITLE
npm rebuild still running

### ISSUE THREAD

**Issue by yanisIk** [mixmaxhq/eb-disable-npm#1]
Hi, 

First of all thanks for this simple and so useful script.
I was using your first version (eb-fix-npm) and now using this because I deploy the app directly with the node_modules.

Unfortunately, it is still running npm rebuild when I deploy my app, is this behavior normal ? 

Thanks

**Comment by wearhere** [comment]
No, that should not be happening. How do you know that is happening?
